jenkins-scripts/lib/make_test_rerun_failed.bash

```diff
@@ -6,7 +6,7 @@ set -e
 # then rerun the failed tests one time and merge the junit results
 # The MAKE_TEST_RERUN_ARGS variable and any arguments to this script
 # hold extra make ARGS, such as -R UNIT_*
-# Note that flaky_junit_merge.py requires lxml python package
+# Note that flaky_junit_merge.py requires lxml python3 package
```

Comment:
Minor comment, but I think this is technically not needed

Reply:
I think you mean that since this is a comment, it doesn't need to change. When I first read it, I thought you were saying the original comment is incorrect

Reply:
at least in Debian/Ubuntu we have different packages for python2/python3 versions of lxml.